src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -1906,7 +1906,7 @@ class C
     static int field;
     static long x { set { System.Console.WriteLine($""setX {value}""); } }
     static string y { get; set; }
-    static ref int z { get { return ref field; } }
+    static ref int z { get { return ref @field; } }
```

Comment:
No need to address in this PR. But, are we considering reporting anything in the following scenario, after `field` feature ships?

```cs
class C
{
    int field;
    public int Prop => field; // doesn't read 'C.field'.
}
```

Essentially some kind of guardrail to try and avoid surprise here when `field` doesn't mean what the user thinks it means in the property accessor. For example, a nudge to declare `int field` as `int @field`, to signal they understand the new behavior in property accessors. (It feels like a warning on the usage of `field` when a symbol with that name is in scope is too cumbersome/hard to suppress.)